galaxyproject/galaxy
# Remove endpoint that would break the tool shed.

I believe this will cause all the tool shed repositories it affects (potentially all of them) to reduce all repositories to a single downloadable changeset.

I could write tests to try to verify it explicitly but that set_repository call causes resetting_all_metadata_on_repository to become False an d that seems to result in this behavior in my unit testing... (admittedly I haven't written an end-to-end test for this though).


## How to test the changes?
(Select all options that apply)
  1. Remove the endpoint, deploy, and see if anyone complains - if they do write some tests and restore it with confidence that it doesn't break... everything.

## License
